libs/sdk-ui-dashboard/src/presentation/widget/common/WidgetScrollablePanel.tsx

```diff
@@ -18,6 +18,7 @@ export const WidgetScrollablePanel: React.FC<IWidgetScrollablePanelProps> = ({ c
 
     const onScroll = debounce(() => {
         if (node?.current) {
+            //TODO: RAIL-4614
             // fireGoodstrapScrollEvent(node.current, window); // TODO: what should this do?
```

Comment:
Come commented code, what we plan to do with it? There is some ticket number, but it's not clear what should happen.

Reply:
I have this ticket on me and will be continue after this will be merged :)